lit_tests/kernel/wave/barriers.py

```diff
@@ -262,4 +262,14 @@ def test_gemm():
 
 if __name__ == "__main__":
     logging.basicConfig(level=logging.DEBUG)
-    unittest.main()
```

Comment:
None of the lit tests are actually using unittest (or pytest), I think, so you can just remove related code and imports entirely.

Reply:
Ah... That would certainly be simpler. Let me try that...

Reply:
That works. Thanks!